tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -440,20 +445,45 @@ private void setTaskFamily(final List<String> listOfEcsTasksBeforeNewOneWasAdded
         }
     }
 
-    public void uploadRunInfo() {
-        runMetrics.setExecutionStatus(getExecutionStatus());
+    private void uploadRunInfo() {
+        extendedGa4GhApi.executionMetricsPost(new ExecutionsRequestBody().addRunExecutionsItem(runMetrics), Partner.AGC.name(), getEntryNameForApi(), version, "generated with tooltester ('run-workflows-through-wes' command)");
```

Comment:
Could also include the environment that the workflow was originally from in the description

Reply:
Decided against after discussion with @kathy-t